### PR TITLE
Don't force people to use our default gitignore file or line-ending normalization

### PR DESCRIPTION
**Pull request by aroben**
Users can still opt-in to these behaviors on a per-repository basis (and GitHub for Windows will encourage them to do so). But setting them as a default for all existing repositories is just too confusing.

### CODE REVIEW

**Comment by haacked** [comment]
:shipit:

File: etc/gitattributes [hunk 1]
```diff
@@ -1,6 +1,3 @@
-# Auto detect text files and perform LF normalization
-* text=auto
-
 # Custom for Visual Studio
```
Comment (tclem) [inline etc/gitattributes:1]: Do we still want these in here?

**Comment by tclem** [comment]
otherwise, :cool:

File: etc/gitattributes (hunk 1 above)
Comment (aroben) [inline etc/gitattributes:1]: There's no harm in them. But I don't know much about them either.

**Pull request merged**